src/origin_sdk/OriginSession.py

```diff
@@ -153,6 +153,43 @@ def __inputs_gql_request_with_loading_handler(
 
         return data
 
+    def __inputs_gql_workbook_status(self, scenario_id: str):
+        request_id = None
+
+        while True:
```

Comment:
This feels a bit risky. Maybe a retry-with-backoff for the http request would be better? This could potentially lock up the SDK if there's something wrong with the workbook request